internal/cli/dialog/aliases.go

```diff
@@ -18,7 +18,7 @@ const (
 	aliasesHelp  = `
 Aliases allow you to call frequently used Git Town commands
 with less typing. For example, if the "git town sync" command
-is aliased, you can call it as "git sync".
+is aliased, you can call it as "git town sync".
```

Comment:
**issue:** This instance should be left unchanged, since it's specifically talking about the aliases feature.

Reply:
Thanks for catching this! 🙏